lib/Doctrine/ODM/PHPCR/Mapping/Driver/XmlDriver.php

```diff
@@ -19,10 +19,10 @@
 
 namespace Doctrine\ODM\PHPCR\Mapping\Driver;
 
-use Doctrine\Common\Persistence\Mapping\ClassMetadata;
-use Doctrine\Common\Persistence\Mapping\Driver\FileDriver;
 use Doctrine\Common\Persistence\Mapping\MappingException as DoctrineMappingException;
```

Comment:
This one looks suspicious like a candidate for a namespace change.

Reply:
Oh right I somehow missed it